src/basic-syntax/compound-types.md

```diff
@@ -39,10 +39,10 @@ Arrays:
 
 * We can use literals to assign values to arrays.
 
-* In the main function, the print statement asks for the debug implementation with the `?` format
-  parameter: `{}` gives the default output, `{:?}` gives the debug output. We
-  could also have used `{a}` and `{a:?}` without specifying the value after the
-  format string.
+* The `println!` macro asks for the debug implementation with the `?` format
```

Comment:
Maybe add a comment on what  `Debug` output means (ie we can get it by adding `#[derive(Debug)]` to the struct definition)

Reply:
I wanted to keep it light here, we don't know anything about traits or so at this point and so I think `#[derive(Debug)]` is a little early. It will appear later in the afternoon.